docs/user-guide/102-policy-syntax.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Policy Syntax"
+tag: "user-guide"
+---
+
+The policy contains the following fields:
+- `capture`: Contains some simple expressions
+to evaluate the current network configuration and store the result at some 
+variables
+- `desiredState`: Contains network state yaml with references to `capture` entries.
+
+__Example:__
+```yaml
+{% include_absolute 'examples/bridge-on-default-gw-dhcp/policy.md' %}
+```
+
+## Capture syntax
+
+The capture is a map of capture entries with an identifier as the key that can 
+be referenced with ```capture.[id]```
+
+The value from the map is a capture entry expression that evaluate the current 
```

Comment:
The map value is...

I think that `evaluate` and `store` need to come with a `s` at the end.